src/main/webapp/WEB-INF/jsp/feedback_success.jsp

```diff
@@ -1,155 +1,23 @@
-.feedback-portlet .icons li {
-  display:inline-block;
-  text-align:center;
-  margin:0px 15px;
-}
-.feedback-portlet .icons li:before {
-  content:"";
-}
-.feedback-portlet .icons ul {
-  padding-left:0;
-  text-align:center;
-  margin-top:20px;
-}
-.feedback-portlet .btn-share-ideas {
-  font-weight:100;
-  font-size:1.4em;
-  background-color:#b70101;
-  padding:15px 50px;
-  margin:10px 0px;
-  color:#fff !important;
-  border-radius:4px;
-  border:0px solid transparent;
-  box-shadow:0px 5px 0px #600;
-  cursor:pointer;
-  transition:0.1s ease-in-out;
-  position:relative;
-}
-.feedback-portlet .btn-share-ideas:hover {
-  transform: translateY(2px);
-  background-color:#900;
-  text-decoration:none;
-  box-shadow:0px 3px 0px #000;
-  transition:0.1s ease-in-out;
-}
-
-.feedback-portlet .general-feedback {
-  border-right:1px solid #b70101;
-  padding:0px 20px;
-  margin-top:20px;
-}
-.feedback-portlet .general-help {
-  padding:20px;
-}
-.feedback-portlet .general-help a {
-  font-size:18px;
-  padding:12px;
-  background-color:#eee;
-  margin-bottom:10px;
-  border-radius:4px;
-  display:block;
-}
-.feedback-portlet .general-help a:hover {
-  text-decoration:none;
-  background-color:#ddd;
-}
-.feedback-portlet .general-help i {
-  font-size:1.3em;
-  margin:0px 20px 0px 10px;
-}
-.feedback-portlet .general-feedback h2,
-.feedback-portlet .general-help h2 {
-  margin:0px 0px 10px;
-  font-size:1.8em;
-  color:#b70101;
-}
-.feedback-portlet .btn-gen-feedback {
-  padding:8px 30px;
-}
-.feedback-portlet .thanks {
-  margin-bottom:150px;
-}
-@media (max-width:768px) {
-  .feedback-portlet .btn-share-ideas {
-    padding:6px 12px;
-  }
-}
-
-</style>
-
 <div class='feedback-portlet'>
   <div class='container-fluid'>
-    <div class='row'>
-      <div class='col-xs-12 my-voice'>
-        <div class="my-voice-title">
-          <span><h1 class="light">MyUW</h1><img src="http://umark.wisc.edu/brand/templates-and-downloads/downloads/web/uwcrest_web_sm_fpo.png"><h1 class="heavy">MyVoice</h1></span>
-        </div>
-        <h2>Tell us how we could improve your MyUW experience.</h2>
-        <div class="icons">
-          <ul>
-            <li><i class="fa fa-lightbulb-o"></i><h2>Share</h2></li>
-            <li><i class="fa fa-comments-o"></i><h2>Discuss</h2></li>
-            <li><i class="fa fa-thumbs-o-up"></i><h2>Vote</h2></li>
-          </ul>
-        </div>
-        <a href="https://myuw.ideascale.com/a/panel.do" target="_blank" class="btn btn-primary btn-large btn-share-ideas">Share your ideas <i class="fa fa-arrow-right"></i></a>
-      </div>
-    </div>
     <div class="row">
-      <div class="col-xs-12 col-sm-6 general-feedback">
-        <h2>Thank you!</h2>
-        <p class="thanks">Thanks for getting in touch.  If you indicated that you wanted us to follow-up with you, we'll be in touch shortly.</p>
+      <div class="col-xs-12 col-sm-6 general-thanks">
+        <div class="info-area">
+            <p>Thanks for your feedback.</p>
```

Comment:
Should there be a link here to give more feedback so that the feedback provider knows how to reset this feedback page?